docs/development.md

```diff
@@ -223,8 +223,10 @@ There are several files used to configure various aspects of the system.
 * `gradle.properties`, `gradle-wrapper.properties`: Contains the Gradle and Gradle wrapper configuration.
 * `package.json`: Contains the front-end third-party dependencies specification, as well as configurations for automated tasks/routines to be run via NPM.
 * `angular.json`: Contains the Angular application configuration.
-* `.travis.yml`: Contains the Travis CI job configuration.
-* `appveyor.yml`: Contains the AppVeyor CI job configuration.
+* `component.yml`: GitHub Action configuration for component tests run on every pull request/push to master or release branch.
```

Comment:
"run on every push to master or release branch" is an additional detail that needs not be mentioned here. It also necessitates more places to be changed if we ever decide to change the practice.